app/controllers/users/omniauth_controller.rb

```diff
@@ -17,4 +24,10 @@ def failure
     flash[:notice] = 'Login failed'
     redirect_to root_path
   end
+
+  private
+
+  def game
+    @game ||= Game.find_by(code: cookies[:code])
```

Comment:
`@game ||= Game.unstarted.find_by(code: cookies[:code])`
and add `unstarted` to 
https://github.com/Vitaliy-Bondarenko/Office_killers/blob/master/app/controllers/games_controller.rb#L15
https://github.com/Vitaliy-Bondarenko/Office_killers/blob/master/app/controllers/api/v1/players_controller.rb#L22